modules/timedrift/timedrift.go

```diff
@@ -106,12 +118,31 @@ func (r *run) Run(in io.Reader) (out string) {
 	}
 	// assume host has synched time and set to false if not true
 	el.IsWithinDrift = true
+
+	//Load ntp servers from /etc/ntp.conf
+	ntpFile, err = os.Open("/etc/ntp.conf")
+	if err != nil {
+		panic(err)
+	}
+	defer ntpFile.Close()
+
+	ntpScan = bufio.NewScanner(ntpFile)
+	for ntpScan.Scan() {
+		ntpLine := strings.TrimSpace(ntpScan.Text())
+		if strings.HasPrefix(ntpLine, "server ") {
+			NtpPool = append(NtpPool, strings.TrimSpace(ntpLine[7:]))
```

Comment:
Looks like the string needs to be split a little further: the format is `server address [options ...]` and we just want the address.
Debian ships with these lines:
```
server 0.debian.pool.ntp.org iburst
server 1.debian.pool.ntp.org iburst
server 2.debian.pool.ntp.org iburst
server 3.debian.pool.ntp.org iburst
```
A `string.Split` that extracts the 2nd entry should do the job.